fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -687,6 +691,8 @@ alterClause
     | columnRenameClause
     | reorderColumnsClause
     | rollupRenameClause
+    | addMaterializedColumnClause
```

Comment:
Merge into existing grammars, no need to add grammars separately. Same as when creating a table, modify columnDesc uniformly